module/cli_controller.py

```diff
@@ -129,7 +100,7 @@ def process(self) -> None:
 
         self.check_input()
 
-    def stop(self) -> None:
```

Comment:
Params is passed to this method and not used?

Reply:
This is because we use it in the `self.global_commands`, all members of this dictionary should be callable with a list variable. Using this dictionary makes creating commands for the CLI itself way easier compared to adding each command as an if statement somewhere.

Reply:
Is this commented somewhere? Didn't read this anywhere.